### ISSUE TITLE
About the dataset

### ISSUE THREAD

**Issue by Rax111** [Janspiry/Image-Super-Resolution-via-Iterative-Refinement#115]
The Json file only mentions FFHQ as the training set and CelebaHQ as the verification set. For example, FFHQ 128 as the training set. Do we need to find the CelebaHQ 128 verification set by ourselves? What is the role of the CelebaHQ512 and 1024 you provided, and then for example 16 or 64 low-resolution images how to get it

**Comment by hzjian123** [comment]
There are only one image in CelebaHQ128 folder in this repo for model validation. I found this leads to unstable PSNR and maybe we need to prepare whole dataset by ourselves?

**Comment by Janspiry** [comment]
A picture in the folder is just an example. The validation set needs to be prepared according to the process in README.
